examples/power_spectra_example.py

```diff
@@ -0,0 +1,43 @@
+"""
+===============================
+POWER SPECTRUM OF A TIMESERIES
+===============================
+
+An example showing how to estimate power spectrum of a TimeSeries
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import pandas as pd 
+from scipy import signal
+
+import sunpy.timeseries
+from sunpy.data.sample import GOES_XRS_TIMESERIES
+
+###############################################################################
+# Let's first load a TimeSeries from sample data
+
+ts = sunpy.timeseries.TimeSeries(GOES_XRS_TIMESERIES)
+
+###############################################################################
+# We now use scipy's periodogram to estimate the power spectra of the first 
+# column of the Timeseries
+
+cols=ts.columns
+freq, spectra = signal.periodogram(ts.data[cols[0]], fs=1)
```

Comment:
Is `fs=1` correct for the data? 

Also I wonder if we could use the false alarm probability in this example as well.

Reply:
Yeah , this feature is covered in Astropy's periodogram . Should we switch over to Astropy's periodogram ?

Reply:
If it works out regarding the version of Astropy it is supported in.

Reply:
Yeah , it's there in Astropy 3.0

Reply:
Well we still have to support Astropy 2 until our next release.